contribute/style-guides/themes.md

```diff
@@ -2,9 +2,16 @@
 
 ## Overview
 
-**Themes are implemented in Typescript.** That's because our goal is to share variables between Grafana Typescript code and SASS files. Theme definitions are located in `packages/grafana-ui/src/themes/[default|dark|light].ts` files. `default.ts` file holds common variables like typography and spacing definitions, while `[light|dark].ts` primarily specify colors used in themes.
+**Themes are implemented in Typescript.** That's because our goal is to share variables between Grafana TypeScript and [Sass](https://sass-lang.com/) code. Theme definitions are located in the following files:
```

Comment:
```suggestion
**Themes are implemented in Typescript.** That's because our goal is to share variables between Grafana TypeScript and [SASS](https://sass-lang.com/) code. Theme definitions are located in the following files:
```

Reply:
"Sass" uses this capitalization. It's a language name like "Java" or "Ruby". Do you mean the the "Sassy CSS" syntax, "SCSS"? 

References:
- https://sass-lang.com/
- https://en.wikipedia.org/wiki/Sass_(stylesheet_language)
- https://www.designernews.co/stories/49077-sass-its-sass-not-sass